### ISSUE TITLE
It's (somehow) possible to disable all axis

### ISSUE THREAD

**Issue by LogicalError** [RadicalCSG/Chisel.Prototype#84]
Somehow it's possible to get into a state where all axi are disabled, but we don't have the buttons visible in the bottom toolbar to re-enabled them.
You can click on the center point of the position gizmo to re-enable all the axi, unfortunately this doesn't seem to get saved somehow

**Comment by jmickle66666666** [comment]
Yeah I don't understand how these settings are changing but it's happening to me too. Can we disable the axis-locking functionality for now? If that's not a trivial change we can just build a viewport settings menu and maybe attach some logging to the variables to see if we can find out how they change.
